openpype/hosts/fusion/plugins/publish/collect_render.py

```diff
@@ -81,12 +88,12 @@ def get_instances(self, context):
                 tilesX=0,
                 tilesY=0,
                 review="review" in instance_families,
-                frameStart=inst.data["frameStart"],
-                frameEnd=inst.data["frameEnd"],
-                handleStart=inst.data["handleStart"],
-                handleEnd=inst.data["handleEnd"],
-                frameStartHandle=inst.data["frameStartHandle"],
-                frameEndHandle=inst.data["frameEndHandle"],
+                frameStart=frame_start,
+                frameEnd=frame_end,
+                handleStart=handle_start,
+                handleEnd=handle_end,
+                frameStartHandle=frame_start_handle,
+                frameEndHandle=frame_end_handle,
```

Comment:
Wasn't the old logic here just fine? Why do we need to _take it out_ from here?

Reply:
Yeah, there was some logic there which was removed.